react-native/setup/setup-and-deploy-new-project-to-staging.mo.md

```diff
@@ -117,4 +112,4 @@ bundle exec fastlane android deploy --env=staging
 
 ## Troubleshooting
 
-If 'Cloning GitHub repo' takes more than 2 minutes: the github servers may be untrusted Trigering a `git clone git@github.com:bamlab/certificates.git` will fix it.
+If 'Cloning GitHub repo' takes more than 2 minutes: the github servers may be untrusted Triggering a `git clone git@github.com:bamlab/certificates.git` will fix it.
```

Comment:
missing '.':

If 'Cloning GitHub repo' takes more than 2 minutes: the github servers may be untrusted.
Triggering a `git clone git@github.com:bamlab/certificates.git` will fix it.

Reply:
Fixed with last commit.